architecture/index.html

```diff
@@ -680,7 +680,7 @@ <h2>
     <p>
       In response to our <a href="#example-request">previous request</a>:
     </p>
-    <pre><code><span class="next">HTTP/1.1 200 OK</span>
+    <pre><code><span class="next">HTTP/2.0 200 OK</span>
```

Comment:
The problem is that HTTP/1.1 is a plain-text response, whereas HTTP/2 is binary.

So in practice, there can ever be a status line that looks like `HTTP/2.0 200 OK`.

Good exam question though 😉